src/resmom/mom_main.c

```diff
@@ -6554,6 +6560,10 @@ finish_loop(time_t waittime)
 		wait_request(1, NULL);
 	}
 #else
+#ifdef PBS_UNDOLR_ENABLED
+	if (sigusr1_flag)
```

Comment:
instead of using a global, why don't we call undolr() directly from catch_sigusr1()?

Reply:
It is unsafe to call directly from signal handler.